### PR TITLE
resolves 526

### PR DESCRIPTION
**Pull request by ClassicSours**
Fixed taber's comments, by removing useless files,  added configs for each component so they render.

radio is erroring with cannot read propery forEach of undefined.

### CODE REVIEW

File: src/bossy-ui/components/form-element/form-element.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  selector: 'bossy-form-element',
+  templateUrl: './form-element.html',
+  styleUrls: ['./form-element.css']
+})
+export class BossyFormElementComponent {
+  // TODO: Make BossyFormElementConfig
+  // https://github.com/buildcom/BossyUI/issues/528
+  // @Input() config: BossyFormElementConfig;
+
+  type = "element";
```
Comment (glawson) [inline src/bossy-ui/components/form-element/form-element.ts]: type: string = 'element'

Note the single quotes too.

**Comment by glawson** [comment]
@ClassicSours: codeship is breaking on build due to:

> ng lint

Warning: The 'no-use-before-declare' rule requires type checking

ERROR: /home/rof/src/github.com/buildcom/BossyUI/src/bossy-ui/components/form-element/form-element.ts[13, 10]: " should be '
ERROR: /home/rof/src/github.com/buildcom/BossyUI/src/bossy-ui/components/form-element/form-element.ts[18, 3]: Implement lifecycle hook interface OnInit for method ngOnInit in class BossyFormElementComponent (https://angular.io/styleguide#style-09-01)
ERROR: /home/rof/src/github.com/buildcom/BossyUI/src/bossy-ui/bossy-ui.module.ts[16, 81]: Missing semicolon
ERROR: /home/rof/src/github.com/buildcom/BossyUI/src/bossy-ui/bossy-ui.module.ts[39, 81]: Missing semicolon
ERROR: /home/rof/src/github.com/buildcom/BossyUI/src/app/app.module.ts[16, 91]: Missing semicolon

File: src/app/app.module.ts [hunk 2]
```diff
@@ -13,11 +13,13 @@ import {ConfigService} from './config.service';
...
 import { ContributingComponent } from './contributing/contributing.component';
 import { HomeComponent } from './home/home.component';
+import {BossyFormElementComponent} from '../bossy-ui/components/form-element/form-element';
 
 const appRoutes: Routes = [
   { path: '', component: HomeComponent },
   { path: 'sandbox', component: SandboxComponent },
-  { path: 'contributing', component: ContributingComponent }
+  { path: 'contributing', component: ContributingComponent },
+  { path: 'element', component: BossyFormElementComponent }
```
Comment (tabersays) [inline src/app/app.module.ts]: Would try removing this new route and add this to sandbox.component.html

<h2>Form element demo</h2>
  <hr>
  <div class="component-main">
    <bossy-form-element> 
  </div>

**Comment by glawson** [comment]
@ClassicSours - fix conflicts

**Comment by glawson** [comment]
closes #526

**Pull request merged**